src/chains/tezos/utils/index.ts

```diff
@@ -0,0 +1,9 @@
+/*!
+ * @ganache/utils
+ *
+ * @copyright Truffle Blockchain Group
+ * @author David Murdoch <david@trufflesuite.com> (https://davidmurdoch.com)
```

Comment:
You can attribute authorship to yourself and your _consensys_ email address for any packages you create, if you want.